app/assets/stylesheets/common.scss

```diff
@@ -2104,6 +2105,11 @@ textarea {
 textarea {
   padding: 5px;
   width: 100%;
+  min-height: 50px;
+}
+
+textarea.comment {
+  resize: vertical;
```

Comment:
I'd prefer for this to apply to all `textarea` since other places also go haywire when they are resized horizontally. Simply moving this line to the `textarea` section immediately above will be enough.

Reply:
Thank you, I just added it with my latest commit. 👍